Application/Frontend/components/FriendsColumn/get_chat_members.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+
+function ChatMembersFinder() {
+  const [channelKey, setChannelKey] = useState('');
+  const [memberIDs, setMemberIDs] = useState<string[]>([]); // Explicitly state that memberIDs is an array of strings
+  const [usernames, setUsernames] = useState<string[]>([]); // Assuming usernames are also strings
+  const [error, setError] = useState('');
+
+  const fetchChatMembers = async () => {
+    setError('');
+    try {
+      const response = await fetch('/api/get_member', { // Adjust with your API endpoint
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ channelKey }),
+      });
+
+      if (!response.ok) {
+        throw new Error('Failed to fetch chat members');    
+      }
+
+      const data = await response.json();
+      if (data.members && data.members.length > 0) {
+        setMemberIDs(data.members);
+        fetchUsernames(data.members); // Fetch usernames after setting member IDs
+      } else {
+        setMemberIDs([]);
```

Comment:
You actually don't need to call any of these setters because they are already initialized with empty arrays by default as you are using React States.